pkg/controller/servicebindingrequest/custom_env_parser.go

```diff
@@ -42,3 +43,10 @@ func (c *CustomEnvParser) Parse() (map[string]interface{}, error) {
 	}
 	return data, nil
 }
+func MarshalToJSON(m interface{}) (string, error) {
```

Comment:
Nit pick -This function is not an exported function. We can have it as `marshalToJson`  to maintain consistency